### PR TITLE
Updates I found when working around

### PR DESCRIPTION
**Pull request by rhyolight**
Also from [the forum](https://discourse.numenta.org/t/bad-links/2334/4?u=rhyolight).

### CODE REVIEW

File: swarm_examples/README.md [hunk 1]
```diff
@@ -141,35 +146,39 @@ The swarm can be started with the similar command but note that the process will
...
 
-The end error was 0.89%, significantly better than before. This means at least one 
-additional field helped.  But which one? At the end, look for a Field Contributions 
-JSON, that looks like this:
+Previous:
+> The end error was 0.89%, significantly better than before. This means at least
+> one additional field helped.  But which one? At the end, look for a Field 
+> Contributions JSON, that looks like this:
+
+I'm not sure what happened here. -- Matt
```
Comment (rhyolight) [inline swarm_examples/README.md]: @subutai I wanted to draw your attention to this. I reran these examples and got different results. You may want to run them on your system and see what you come up with.

Comment (subutai) [inline swarm_examples/README.md]: The numbers have changed, but I still got something reasonable: 

> Field Contributions:
> {   u'metric1': 0.0,
>     u'metric2': 49.70003462701029,
>     u'metric3': -72.71102127188551,
>     u'metric4': -95.16160693185664,
>     u'metric5': -95.16160693185664}
> 

and 

> Best results on the optimization metric multiStepBestPredictions:multiStep:errorMetric='altMAPE':steps=[1]:window=1000:field=metric1 (maximize=False):
> [83] Experiment _NupicModelInfo(jobID=1002, modelID=1385, status=completed, completionReason=eof, updateCounter=22, numRecords=1500) (modelParams|tmParams|minThreshold_11.modelParams|tmParams|activationThreshold_16.modelParams|tmParams|pamLength_5.modelParams|clParams|alpha_0.0999932803103.modelParams|sensorParams|encoders|metric2:n_521.modelParams|sensorParams|encoders|metric1:n_465.modelParams|spParams|synPermInactiveDec_0.00170177618031):
>   multiStepBestPredictions:multiStep:errorMetric='altMAPE':steps=[1]:window=1000:field=metric1:    1.07995291262

Comment (subutai) [inline swarm_examples/README.md]: with basic_search_def.json I got an error of 2.15, so the above error of 1.08 is still a significant improvement.

Comment (rhyolight) [inline swarm_examples/README.md]: @subutai Ok great. Ready to merge then?

Comment (subutai) [inline swarm_examples/README.md]: The code and intstruction changes look good (thanks!) but the numbers reported in this PR, including the field contributions,  don't look correct and don't match my results.

Comment (rhyolight) [inline swarm_examples/README.md]: I see. I ran them several months ago. I'll run again and confirm.

**Comment by rhyolight** [comment]
Ok @subutai. I re-ran with NuPIC 0.6.0 and got reasonable numbers. README is appropriately updated. Ready for re-review.

File: swarm_examples/README.md [hunk 2]
```diff
@@ -3,18 +3,25 @@ Introduction
 
 This is an example demonstrating swarming for prediction and anomaly detection
 models. The example also demonstrates using multiple fields. We assume you have
-read the following wiki page:
+read the following page:
 
-https://github.com/numenta/nupic/wiki/Running-Swarms
+http://nupic.docs.numenta.org/0.7.0.dev0/guides/swarming/running.html
```
Comment (subutai) [inline swarm_examples/README.md]: Should we use `latest` in the URL instead of 0.7.0?

File: swarm_examples/README.md [hunk 3]
```diff
@@ -3,18 +3,25 @@ Introduction
...
+Sine waves are actually not ideal for the CLA. It is a single continuously
 repeating pattern with no clear beginning and end. Please see an email
 from Jeff Hawkin on this topic:
 
 http://lists.numenta.org/pipermail/nupic_lists.numenta.org/2013-June/000327.html
 
+Assumptions
+===========
+
+You have [NuPIC](http://github.com/numenta/nupic) 0.6.0 installed and the `NUPIC`
```
Comment (subutai) [inline swarm_examples/README.md]: It says 0.7.0 in the above URL... which version do we need??

File: swarm_examples/README.md [hunk 4]
```diff
@@ -3,18 +3,25 @@ Introduction
...
+environment variable set to the repository path.
+
+    pip install nupic==0.6.0
```
Comment (subutai) [inline swarm_examples/README.md]: Maybe leave out NuPIC installation instructions and just put in a link to NuPIC?

File: swarm_examples/README.md [hunk 5]
```diff
@@ -3,18 +3,25 @@ Introduction
...
-read the following wiki page:
+read the following page:
 
-https://github.com/numenta/nupic/wiki/Running-Swarms
+http://nupic.docs.numenta.org/0.7.0.dev0/guides/swarming/running.html
 
 DISCLAIMER: we are using sine waves below to demonstrate some of the points
 because this seems to be one of the most popular first tasks that people try.
-Sine waves are actually not ideal for the CLA. It is a single continuously 
+Sine waves are actually not ideal for the CLA. It is a single continuously
```
Comment (subutai) [inline swarm_examples/README.md]: replace CLA with "for HTM sequence learning"

File: swarm_examples/README.md [hunk 6]
```diff
@@ -42,20 +49,24 @@ metric4 is going to be a perfect predictor for metric5 at the next time step.
...
 
 
 
 Prediction vs Anomaly Detection
 ===============================
 
-The CLA basically does anomaly detection by doing prediction and looking for unpredicted situations.
-All of the examples below use `inferenceType` = `TemporalAnomaly` in the JSON files. The CSV files contain 
-predictions as well as anomaly scores.
+The CLA basically does anomaly detection by doing prediction and looking for
```
Comment (subutai) [inline swarm_examples/README.md]: replace "the CLA" with "HTM sequence learning"

**Review by subutai — APPROVED** [review]
A few minor requests - overall looks great!  Thanks for doing this!

File: swarm_examples/README.md (hunk 3 above)
Comment (rhyolight) [inline swarm_examples/README.md]: The only place those docs exist (until we release 0.7.0) is at that dev URL. Later we can update to "stable" docs.

Comment (subutai) [inline swarm_examples/README.md]: ok, makes sense.

**Comment by rhyolight** [comment]
@subutai Addressed your comments, ready to merge.

**Pull request merged**